File: JAVA/문제_2910번.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.ArrayList;
import java.util.Collections;
import java.util.Comparator;
import java.util.HashMap;
import java.util.List;
import java.util.Map;
import java.util.Map.Entry;
import java.util.LinkedHashMap;

public class Main{
    public static void main(String []args) throws IOException{
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		Map<Integer,Integer> m = new LinkedHashMap();
		int n , c;
		String input[] = br.readLine().split(" ");
		n = Integer.parseInt(input[0]);
		c = Integer.parseInt(input[1]);
		int num[] = new int[n + 1];
		String input2[] = br.readLine().split(" ");
		for(int i = 0;i<input2.length;i++) {
			num[i] = Integer.parseInt(input2[i]);
			if(m.get(num[i]) == null)
				m.put(num[i], 1);
			else
				m.put(num[i], m.get(num[i]) + 1);
		}
		List<Entry<Integer,Integer>> li = new ArrayList<Entry<Integer, Integer>>(m.entrySet());
		
		Collections.sort(li , new Comparator<Entry<Integer, Integer>>(){
			
			public int compare(Entry<Integer, Integer> obj1, Entry<Integer, Integer> obj2) {
				return obj2.getValue().compareTo(obj1.getValue());
			}
		});
		
		for(Entry<Integer,Integer> entry: li) {
			for(int j = 0; j<entry.getValue();j++) {
				System.out.print(entry.getKey() + " ");
			}
		}
	}
}

File: JavaScript/문제_2910번.js
let input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');
input[0] = input[0].split(' ');
let num = input[1].split(' ');
let n = input[0][0];
let c = input[0][1];
let m = new Map();
for (let i = 0; i < num.length; i++) {
    if (!m.has(num[i]))
        m.set(num[i], 1);
    else
        m.set(num[i], m.get(num[i]) + 1);
}
let arr = [...m];
arr.sort((o1, o2) => {
    return o2[1] - o1[1];
})
let result = [];
for (let i = 0; i < arr.length; i++) {
    for (let j = 0; j < arr[i][1]; j++) {
        result.push(+arr[i][0]);
    }
}
console.log(result.join(' '));